cobigen-templates/templates-devon4j/src/main/templates/crud_angular_client_app/templates/src/app/${variables.etoName#lower_case}/${variables.etoName#lower_case}-dialog/${variables.etoName#lower_case}-dialog.component.ts.ftl

```diff
@@ -11,7 +11,7 @@ export class ${variables.etoName?cap_first}DialogComponent {
   title: string;
   items: any = {
   <#list pojo.fields as field>
-    ${field.name}: '',
+    ${field.name?lower_case}: '',
```

Comment:
If you use `?lower_case` this means that the whole word will be lower case. Are you sure about this?

For example, imagine that my field is `streetNumber` following camel case standard it should not be converted to `streetnumber`.

I would use `uncap_first` instead.

Reply:
One of the fields in the application is `AGE`. Using `uncap_first` makes it `aGE`. How about if I chain `capitalize` and `uncap_first` together?

Reply:
> One of the fields in the application is AGE

This is the problem that caused the back-end to fail. If a user defines a Java Entity, and it does not comply with naming conventions (it should be `age`), then we should not fix his mistakes.

Our developers need to follow naming conventions, no one should define a field as `AGE`.

Reply:
So I'll proceed with replacing `lower_case` with `uncap_first` wherever required. Correct?

Reply:
Correct